src/app/services/views/ServiceDetailTabsView.vue

```diff
@@ -51,10 +51,30 @@
         <LoadingBlock v-else-if="data === undefined" />
 
         <template v-else>
-          <NavTabs
-            class="route-service-detail-view-tabs"
-            :tabs="getNavTabs(data)"
-          />
+          <NavTabs :active-route-name="route.active?.name">
+            <template
+              v-for="{ name } in route.children.filter(({ name }) => {
+                if (data.serviceType !== 'external' && name === 'service-config-view') {
+                  return false
+                }
+
+                if (data.serviceType === 'external' && name === 'service-data-plane-proxies-view') {
+                  return false
+                }
+
+                return true
+              })"
```

Comment:
**Comment**: This needing to take place in the template is a downside with this approach because we can’t cache the result using `computed`. Seeing as the need for filtering will go away with https://github.com/kumahq/kuma-gui/pull/2190, it’s not a big problem though.